test/runtests.jl

```diff
@@ -0,0 +1,6 @@
+using WinRPM
+using Test
+WinRPM.install("gcc", yes = true)
```

Comment:
this has the same problem as https://github.com/JuliaPackaging/WinRPM.jl/pull/95#issuecomment-273035617 - it should clean up after itself instead of making a mess of the user's environment